### ISSUE TITLE
Do not generate documents if there are none

### ISSUE THREAD

**Issue by ddgenome** [dotansimha/graphql-code-generator#167]
Currently if you provide a glob pattern on the command line and it matches no documents, you get an error.

```
> gql-gen --file node_modules/gql-client/graph/schema.json --template typescript --no-schema --out src/typings '{graphql,src}/**/*.graphql'

Loading GraphQL Introspection from file: node_modules/gql-client/graph/schema.json...
Error: Unable to locate files matching glob definition: {graphql,src}/**/*.graphql
```

It seems reasonable that in the case of no matches, code generation could proceed as if there were no documents provided. This would allow using the same package scripts for building projects both with and without GraphQL files (some people prefer to put the GraphQL in strings within the TypeScript source code).

Is there any interest in allowing this behavior? I'd be happy to implement but would like some feedback on whether it would be a welcome change or not and how you would like it implemented, e.g., is it OK to change the default behavior or would you prefer a command-line option like `--allow-empty-glob` or `--documents-if-any` be added.

**Comment by ddgenome** [comment]
I guess that's a "no".

**Comment by dotansimha** [comment]
@ddgenome actually I missed it. PRs are welcome :)

**Comment by dotansimha** [comment]
Fixed in 0.8.17 :)
